change/@microsoft-teams-js-f64aa406-62eb-4eb3-92bd-760dfc43fe75.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "update the raiseHandStateChangedEvent interface to make the error undefinable instead of nullable",
```

Comment:
Please update this to "Renamed `IRaiseHandStateChangedEvent` interface to `IRaiseHandStateChangedEventData` and changed the error so it can be assigned `undefined` rather than `null`"